ingest/rules/curate.smk

```diff
@@ -116,28 +116,33 @@ rule curate:
                 --output-seq-field {params.sequence_field} ) 2>> {log}
         """
 
-
-rule replace_strain_names:
+rule subset_curated_metadata_columns:
     input:
-        metadata="data/all_metadata.tsv",
-        strains = "data/strain-names.tsv"
+        metadata="data/metadata_curated.tsv",
     output:
-        metadata="data/all_metadata_with_strains.tsv",
+        metadata="data/metadata_subset.tsv",
+    params:
+        metadata_fields=",".join(config["curate"]["metadata_columns"]),
     shell:
-        """
-        tsv-select -H --exclude strain {input.metadata} | \
-        tsv-join -H --filter-file {input.strains} --key-fields accession --append-fields strain > {output.metadata}
+        r"""
+        tsv-select -H -f {params.metadata_fields} \
+            {input.metadata} > {output.metadata}
         """
 
-rule subset_metadata:
+rule merge_metadata:
     input:
-        metadata="data/all_metadata_with_strains.tsv",
+        strain="data/strain-names.tsv",
+        main="data/metadata_subset.tsv",
+        segments=expand("data/nextclade_{segment}_summary.tsv", segment=segments),
     output:
-        metadata="results/all/metadata.tsv",
+        metadata="data/metadata_merged.tsv",
     params:
-        metadata_fields=",".join(config["curate"]["metadata_columns"]),
+        # augur merge requires NAME=FILEPATH argments, so we transform the inputs here:
+        segments = lambda w,input: " ".join([f"s_{idx}={s}" for idx,s in enumerate(input.segments)])
     shell:
-        """
-        tsv-select -H -f {params.metadata_fields} \
-            {input.metadata} > {output.metadata}
+        r"""
+        augur merge \
+            --metadata strains={input.strain} main={input.main} {params.segments} \
+            --metadata-id-columns accession \
+            --output-metadata {output.metadata}
```

Comment:
_non-blocking_

You might want to add the `--no-source-columns` flag to exclude the 15 `__source*` columns in the output.

Reply:
This is now the default behaviour (may have happened while you were away?)

Reply:
But i'll add it to be explicit

Reply:
Ah right, it changed with [Augur 26.0.0](https://github.com/nextstrain/augur/releases/tag/26.0.0). Reminder for me to update my runtime.

Reply:
Added `--no-source-columns` via force push